### ISSUE TITLE
Testsuite fails on Expansion: #t0088: Value Expansion does not expand native values, such as booleans, to a node object

### ISSUE THREAD

**Issue by oohlaf** [digitalbazaar/pyld#71]
Hi,

The testsuite fails on (Python3.5):
```

Expansion: #t0088: Value Expansion does not expand native values, such as booleans, to a node object ...

Traceback (most recent call last):
  File "tests/runtests.py", line 228, in runTest
    result = getattr(jsonld, fn)(*params)
  File "tests/../lib/pyld/jsonld.py", line 158, in expand
    return JsonLdProcessor().expand(input_, options)
  File "tests/../lib/pyld/jsonld.py", line 876, in expand
    expanded = self._expand(active_ctx, None, document, options, False)
  File "tests/../lib/pyld/jsonld.py", line 2173, in _expand
    active_ctx, key, value, options, inside_list=False)
  File "tests/../lib/pyld/jsonld.py", line 1974, in _expand
    active_ctx, active_property, e, options, inside_list)
  File "tests/../lib/pyld/jsonld.py", line 1998, in _expand
    return self._expand_value(active_ctx, active_property, element)
  File "tests/../lib/pyld/jsonld.py", line 2667, in _expand_value
    return {'@id': self._expand_iri(active_ctx, value, base=True)}
  File "tests/../lib/pyld/jsonld.py", line 3998, in _expand_iri
    if ':' in value:
TypeError: argument of type 'bool' is not iterable
ERROR

======================================================================
ERROR: Expansion: https://json-ld.org/test-suite/tests/expand-manifest.jsonld#t0088: Value Expansion does not expand native values, such as booleans, to a node object
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/runtests.py", line 240, in runTest
    raise e
  File "tests/runtests.py", line 228, in runTest
    result = getattr(jsonld, fn)(*params)
  File "tests/../lib/pyld/jsonld.py", line 158, in expand
    return JsonLdProcessor().expand(input_, options)
  File "tests/../lib/pyld/jsonld.py", line 876, in expand
    expanded = self._expand(active_ctx, None, document, options, False)
  File "tests/../lib/pyld/jsonld.py", line 2173, in _expand
    active_ctx, key, value, options, inside_list=False)
  File "tests/../lib/pyld/jsonld.py", line 1974, in _expand
    active_ctx, active_property, e, options, inside_list)
  File "tests/../lib/pyld/jsonld.py", line 1998, in _expand
    return self._expand_value(active_ctx, active_property, element)
  File "tests/../lib/pyld/jsonld.py", line 2667, in _expand_value
    return {'@id': self._expand_iri(active_ctx, value, base=True)}
  File "tests/../lib/pyld/jsonld.py", line 3998, in _expand_iri
    if ':' in value:
TypeError: argument of type 'bool' is not iterable

----------------------------------------------------------------------
Ran 565 tests in 9.384s

FAILED (errors=1, skipped=142)
```

**Comment by oohlaf** [comment]
This might be related to issue #53

**Comment by davidlehn** [comment]
There are some recently added tests in the test suite that cover JSON-LD 1.0 behavior.  pyld hasn't been updated to deal with everything yet.  Effort right now is on the 1.1 spec and jsonld.js.  Once that's further along all the recent refactoring and updates in jsonld.js will move over to pyld.  If someone wants to fix this particular issue, then as always, patches are welcome.

**Comment by oohlaf** [comment]
ah, ok. I tried a stab at updating it, but I only introduced more failures in other test cases ;-) I'm not yet familiar enough with the code. I would really like a version that uses aiohttp in stead of requests, but I think it needs more than just setting `set_document_loader`.

**Comment by oohlaf** [comment]
Closing, as this is solved in PR #76 and the aiohttp stuff is in #73 or #77
